WordPress/WordPressTest/ReaderTopicServiceTest.swift

```diff
@@ -12,72 +12,71 @@ class ReaderTopicSwiftTest: XCTestCase {
 
     override func setUp() {
         super.setUp()
-        testContextManager = TestContextManager()
```

Comment:
Doesn't this change mean we're no longer overriding the shared instance of the context manager?

Reply:
I am not sure I understand the question. We would be creating a new instance for every test, via the static initialiser, and nullifying that instance on tearDown. Although now that you mention it, tests within this suite could pollute each other. I'll push an update. Thanks!